runtime/help/options.md

```diff
@@ -398,6 +398,11 @@ Here are the available options:
 
 	default value: `4`
 
+* `detectlimit`: if this is not set to 0, it will limit the automatic parsed
+   lines of files to the given value.
```

Comment:
I think this description is unclear.
* Most importantly for the user, it does not make it clear that it affects the accuracy of file type detection.
* In particular, as you probably know, parsing lines for filetype detection is not the only kind of parsing done by micro. Once file type is detected, micro parses the entire file for syntax highlighting (which is BTW far more expensive). So if mention "parsing lines of files", need to make it clear that it is about parsing for filetype detection.
* Also worth clarifying that it is about the *first* lines of a file.



While we're at it, how about a more explanatory / less ambiguous name for the option? E.g. maybe `ftdetectlimit`?

Reply:
> While we're at it, how about a more explanatory / less ambiguous name for the option? E.g. maybe `ftdetectlimit`?

This was a conscious decision, because I already kept in mind the following [proposal](https://github.com/zyedidia/micro/pull/2819#issuecomment-1583354417). This can then be covered by the same option, since the parsing/detection can take place in the same cycle(s) of the first lines and as you already stated we don't want to create too many options (for more or less the same intention).